app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  def new
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(comment_params)
+    @comment.author = @user
+    if @comment.save
+      flash[:success] = 'Comment created successfully'
+      redirect_to user_post_path(@user, @post)
+    else
+      flash.now[:errors] = 'Error: Comment could not be saved'
+      render :new
+    end
+  end
+
```

Comment:
- [ ] Please ensure that you fix this error that I get after creating a comment :pray: 
  
  ![image](https://github.com/BANCUNGUYE66/Blog-app/assets/92303891/ce974d87-d3e5-4ee4-a24e-0ef84424cf89)